### PR TITLE
Fixes #33504 - align hostname title in host details page

### PR DESCRIPTION
**Pull request by amirfefer**
#### Before: 
Screen Shot 2021-10-03 at 18 26 20

#### After (long hostname):
Screen Shot 2021-10-03 at 18 13 06

#### After (short hostname):
Screen Shot 2021-10-03 at 18 10 40

### CODE REVIEW

**Comment by ekohl** [comment]
There appears to be quite a bit of whitespace remaining. Can't that be used instead when there is space?

There's also a merge conflict now.

**Comment by ezr-ondrej** [comment]
Can't the badges be in the same GridItem so it spreads the space together with the title?

**Comment by amirfefer** [comment]
@ezr-ondrej , @ekohl  - Thanks,
How about this? 

Screen Shot 2021-10-05 at 19 42 33

Screen Shot 2021-10-05 at 19 54 21

File: webpack/assets/javascripts/react_app/components/HostDetails/index.js [hunk 1]
```diff
@@ -78,21 +81,39 @@ const HostDetails = ({
...
+                          </Title>
+                        )}
+                      </SkeletonLoader>
+                    </div>
+                    <Split style={{ display: 'inline-flex' }} hasGutter>
+                      <SplitItem>
+                        <HostGlobalStatus hostName={id} />
+                      </SplitItem>
+                      <SplitItem>
+                        <Badge> {response?.operatingsystem_name}</Badge>
```
Comment (ezr-ondrej) [inline webpack/assets/javascripts/react_app/components/HostDetails/index.js:106]: Probably uneccessary, but doesn't hurt, so I'll merge unless I find something else :)
```suggestion
                        <Badge> {response.operatingsystem_name}</Badge>
```

**Review by ezr-ondrej — APPROVED** [review]
LGTM :+1:

**Comment by ezr-ondrej** [comment]
CP 3.0: #8842

**Pull request merged**